src/webapp/src/auth/Login.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const Login = () => (
```

Comment:
These are the authentication options page to me. Perhaps we can call it `AuthenticationOptions`?

Reply:
Done